api/python/cell_census/scripts/formatting.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/env bash
+
+# Script run by GitHub Action formatting.yml.
+# Dependencies are in `requirements-dev.txt`
+
+python -m black --check --diff .
+python -m isort --check --diff .
+python -m flake8 .
+python -m mypy --python-version 3.7 .
```

Comment:
Could add an arg/flag to _not_ run `--check --diff`, thus making the linting fixes automatically for local use (which can still be reviewed by engineer). I'm happy to add that later as follow-on change.

Reply:
Mmm...I would personally not use that flag (too much magic), but I'm fine if you want to add it later, with a default of "off" :-)